am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetAccessorsListIntegrationTest.java

```diff
@@ -27,7 +28,8 @@ void setUp() {
     @SuppressWarnings("LineLength")
     void ifUserHasAccessShouldReturnUserIds() {
         service.grantExplicitResourceAccess(createGrantForWholeDocument(resourceId, ACCESSOR_ID, READ_PERMISSION));
-        service.grantExplicitResourceAccess(createGrantForWholeDocument(resourceId, OTHER_ACCESSOR_ID, READ_PERMISSION));
```

Comment:
Revert

Reply:
👍

Reply:
I asked for revert, not to remove that call. I simply do not want to see changes in this class as they are irrelevant.